docs/custom.css

```
@@ -1178,6 +1178,15 @@ article p {
   border-bottom: 0.5px solid var(--border) !important;
 }
 
+html.plugin-id-integrations.docs-doc-id-index {
+  div[data-name='doc-breadcrumbs-copy'] {
+    display: none;
+  }
+  h1 {
+    margin-top: 2rem;
+  }
```

Stylelint (17.14.0)

[error] Expected empty line before rule (rule-empty-line-before)

(rule-empty-line-before)